### PR TITLE
Update the README with some Release goodies

### PR DESCRIPTION
**Pull request by brandonb927**
Adds a section about running the binary with a release, and cleans up some formatting stuff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,86 +6,126 @@
...
-For more information about the engine:
+_Protip_: If you want to create and run a game in your browser (provided you have the [`board`](https://github.com/battlesnakeio/board) setup and running, and have [`jq`](https://stedolan.github.io/jq/) installed) you can run a one-liner with:
 
-`engine --help`
+```shell
+ENGINE_URL=http://localhost:3005
+./engine create -c ~/.snake-config.json \
+  | jq --raw-output ".ID" \
+  | xargs -I {} sh -c \
+      "open -a \"/Applications/Google Chrome.app/Contents/MacOS/Google Chrome\" \
```
Comment (dlsteuer) [inline README.md]: obviously this is mac specific, so i'm a bit torn on it. I want to have a --dev mode in the engine, so you can just do something like `engine --dev` which takes care of all of this.

File: README.md [hunk 2]
```diff
@@ -6,86 +6,126 @@
...
+```
 
 ## Backend configuration
 
 Storage options:
 
-* `inmem` - This is the default. All game data is erased when the engine restarts.
-* `file` - Stores one file per game. Games can be resumed or replayed after restart.
+- `inmem` - This is the default. All game data is erased when the engine restarts.
+- `file` - Stores one file per game. Games can be resumed or replayed after restart.
```
Comment (dlsteuer) [inline README.md]: should add `redis` here too

File: README.md (hunk 1 above)
Comment (brandonb927) [reply README.md]: Oof yeah hadn't realized the mac specificness here... I'm wondering if there's a better way to demonstrate this then?

**Comment by brandonb927** [comment]
Thanks @dlsteuer!

**Pull request merged**